tests/unit/stall_detector_test.cc

```diff
@@ -25,7 +25,7 @@
 #include <seastar/util/later.hh>
 #include <seastar/testing/test_case.hh>
 #include <seastar/testing/thread_test_case.hh>
-#include <../../src/core/stall_detector.hh>
+#include <core/stall_detector.hh>
```

Comment:
Does this mean we have the src directory listed as a system header? I think that's wrong.

Alternative: move stall_detector.hh to include/seastar/core/internal/ and include it from there. It's lazy, but we shouldn't rely on src/ being a system include directory.

Reply:
@avikivity fixed as suggested and repushed.